docs/components/content/examples/FormExampleElements.vue

```diff
@@ -44,11 +44,11 @@ const schema = z.object({
 
 type Schema = z.infer<typeof schema>
 
-const form = ref<Form<Schema>>()
+const form = ref()
```

Comment:
Why remove the type here?

Reply:
It's no longer needed since the example does not rely on `form.value.validate` now that validation happens automatically on submit. It was required on previous examples to get the right return type.

The ref itself is also useless, I'll remove it.

Reply:
Maybe to type the `clear` method?